.github/pull_request_template.md

```diff
@@ -7,17 +7,15 @@ If your pull request closes a GitHub issue, then set its number below.
 
 Fixes # (issue)
 
-## Type of change
-
-Please mark the options that are relevant.
-
-- [ ] Added/Modified tutorials
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 
 ## Checklist:
 
 * [ ] I have added tests for my changes
-* [ ] I did follow the [contribution guidelines](https://github.com/OpenMined/PySyft/blob/master/CONTRIBUTING.md)
+
+* [ ] I have Parametrized my test.
```

Comment:
I think we should remove those checks.
This check should already be done by ```flake8``` or ```black```, right?

I think those stuff should already be caught if we merge [this PR](https://github.com/OpenMined/PySyft/pull/3434)